src/System.Reflection.Context/src/Configurations.props

```diff
@@ -1,11 +1,9 @@
 ﻿<Project>
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
       netstandard;
       netstandard1.1;
-      uap;
-      _netfx;
+      netstandard2.1;
```

Comment:
I believe you still want the _netfx configuration, because netstandard is still one of the configs so that will be built in the netfx vertical now.

Reply:
net45 is marked as inbox in the package, means no assets that we build here will be used on .NET Framework. Therefore we don't need to test it.